module/x/gravity/keeper/contract_call.go

```diff
@@ -64,3 +66,19 @@ func (k Keeper) contractCallExecuted(ctx sdk.Context, invalidationScope []byte,
 
 	k.CompleteOutgoingTx(ctx, completedCallTx)
 }
+
+// orderContractCallsByAddressAndNonceAscending sorts a slice of contract calls by address and nonce in ascending order
+func orderContractCallsByAddressAndNonceAscending(calls []*types.ContractCallTx) []*types.ContractCallTx {
+	sort.SliceStable(calls, func(i, j int) bool {
+		// Compare the addresses first
```

Comment:
Wouldn't any sorting of strictly increasing or equal nonce values provide the right order? Why care about the address?